src/scss/grommet-core/_objects.box.scss

```diff
@@ -349,3 +349,24 @@
 .box--clickable {
   cursor: pointer;
 }
+
+.box--size {
+  .paragraph {
+    width: 100%;
+  }
+
+  &-small {
+    width: $inuit-base-spacing-unit * 24;
```

Comment:
Let's make these variables in _settings.defaults.scss and then re-use the same variables in the sizing for Image, Video, Quote, Sidebar, etc.